### PR TITLE
Texture 3 support

### PR DESCRIPTION
**Pull request by foxware00**

### CODE REVIEW

**Review by dangthaison91 — APPROVED** [review]
LGTM, Thanks @foxware00

File: RxASDataSources.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
      s.name = 'RxASDataSources'
-     s.version = '0.5.0'
+     s.version = '0.6.0'
```
Comment (dangthaison91) [inline RxASDataSources.podspec]: Should we bump version to 1.0 to follow version sematic also supporting new Texture version

**Comment by dangthaison91** [comment]
Could you help to rebase your PR @foxware00 ?

**Comment by foxware00** [comment]
> Could you help to rebase your PR @foxware00 ?

Sorry @dangthaison91 I was on leave last week. What's the problem with rebasing? It looks like it should merge just fine.

As for changing it to version `1.0` that's up to you really. Happy to change to `1.0.0` to keep it following semantic versioning

**Comment by dangthaison91** [comment]
> As for changing it to version `1.0` that's up to you really. Happy to change to `1.0.0` to keep it following semantic versioning

I think we should do it, could you help @foxware00 ?

**Comment by foxware00** [comment]
@dangthaison91 Bumped to 1.0.0

**Comment by dangthaison91** [comment]
Thanks @foxware00

**Pull request merged**